ismatec/util.py

```diff
@@ -14,21 +18,22 @@
 
 
 class Communicator(threading.Thread):
-    """Class representing the hardware interface to the Ismatec Reglo ICC peristaltic pump.
```

Comment:
Pet peeve: of course it's a class.

I think of the original purposes of docstrings when I write them.
1. People who want to use your code without knowing how it works at all
2. People who are jumping in to improve the code and need to know how it works

For 1, they'll read the top line of the docstring and then try it out. It should be concise and not leak out implementation details.

For 2, you should basically paste all the "why" behind your function in lines 3+. They'll be reading this in the hope of not having to dig into the manual.